plugin/zoom/lib/API/JWTClient.php

```diff
@@ -0,0 +1,347 @@
+<?php
+/* For licensing terms, see /license.txt */
+
+namespace Chamilo\PluginBundle\Zoom\API;
+
+use Exception;
+use Firebase\JWT\JWT;
+
+/**
+ * Class JWTClient.
+ *
+ * @see https://marketplace.zoom.us/docs/api-reference/zoom-api
+ *
+ * @package Chamilo\PluginBundle\Zoom
+ */
+class JWTClient
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class JWTClient has a coupling between objects value of 14. Consider to reduce the number of dependencies under 13.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5625225)